generated/explanations/euler.md

```diff
@@ -0,0 +1,62 @@
+## Header
+This is the course header. This will be added on top of every page. Go to [DoDAO.io](https://www.dodao.io) to know more.
+
+ ---
+ 
+ ## Euler
+ 
+ **About Euler**        
+
+### What is Euler 
+Euler is a non-custodial DeFi protocol that allows users to lend and borrow and the main factor that differentiates Euler form other trusted protocols like Aave and Compound is that,  in Euler any asset that has a WETH pair on Uniswap v3 can be added as a lending market on Euler by anyone straight away. There is a  significant unmet demand for lending and borrowing  the long tail of crypto assets which are not listed on Aave or Compound. Euler is managed by holders of a protocol native governance token called Euler Governance Token (EUL). 
+
+### Asset Tiers
+Listing multiple assets which are not necessarily mainstream comes with risk as their price can fluctuate a lot. Euler mitigates this risk by using different types of  pools which can/cannot be isolated from other assets.  This limits the exposure in case there are any bad debts. Another concept adopted by Euler is that they consider the risk factor of both the collateral and the borrowed asset to come up with the maximum amount that can be borrowed for the said collateral-liability asset pair called **risk-adjusted collateral**.   For example, in borrowing XYZ with a collateral factor of 0.5, say you are using $1000 USDC with a collateral factor of 0.9, the maximum amount borrowable would come out ot be 1000*0.5*0.9 = $450 worth of XYZ as opposed to the usual 1000*0.9 that we are used to. 
+* Isolation-tier -** ** assets are available for ordinary lending and borrowing, but they cannot be used as collateral to borrow other assets, and they can only be borrowed in isolation. What this means is that they cannot be borrowed alongside other assets using the same pool of collateral.  * Cross-tier -** ** assets are available for ordinary lending and borrowing, and cannot be used as collateral to borrow other assets, but they can be borrowed alongside other assets. For example, if a user has USDC and DAI as collateral, and they want to borrow cross-tier assets ABC and XYZ, then they can do so from a single account on Euler. * Collateral-tier - assets are available for ordinary lending and borrowing, cross-borrowing, and they can be used as collateral. For example, a user can deposit collateral assets DAI and USDC, and use them to borrow collateral assets UNI and LINK, all from a single account.
```

Comment:
Improve the formatting.

Do you cross check the formatting in the generated markdown files?

https://dodao.academy/course/file/view/lending-and-borrowing-in-defi/euler/explanation/about-euler